mods/cnc/mod.yaml

```diff
@@ -196,7 +196,7 @@ SupportsMapsFrom: cnc
 
 SoundFormats: Aud, Wav
 
-SpriteFormats: ShpTD, TmpTD, ShpTS, TmpRA
+SpriteFormats: ShpTD, TmpTD, ShpTS, TmpRA, PngSheet
```

Comment:
I'd say lets call this just Png / PngLoader. None

Reply:
The reason i took a different name, was that a class named PngLoader already existed. I'll recheck that.

Reply:
I like the idea of the name making it clear that this is parsing specially formatted pngs, and not any old image.  How about `AtlasedPng` or similar?

Reply:
To be honest, im open for anything here ;) Also i just updated the PR with support for particular frame information as we have spoken about. And ofcourse added a funny little example sheet which replaces cnc e1 soldiers :)

Reply:
In my opinion PngSheet is fine. We're not using 'Loader' suffix on "end user" yaml side for any other formats and 'Atlased' is a word most non-native english speakers probably haven't come across ever before.

Reply:
While doing changes to this PR, PngLoader no longer exists. So in theory Png would be free as a name again, but as long as there is no demand for that, ill keep it PngSheet because with the metadata its basicaly not a standard png anymore.